e2e/page-objects/onboarding.ts

```diff
@@ -10,9 +10,11 @@ export default class Onboarding {
 	}
 
 	get heading() {
-		return this.locator.getByRole( 'heading', {
-			name: /Connect to your WordPress.com account|Connect your WordPress.com account/,
-		} );
+		return this.locator.getByTestId( 'onboarding-welcome-title' ).or(
```

Comment:
Not something that should be done, I am just thinking for the future cases - are there some disadvantages of using `data-testid` vs roles 🤔 Interesting your thoughts why you decided to switch to testid?